### ISSUE TITLE
[CLOSED] pathing issues

### ISSUE THREAD

**Issue by netherspite** [netherspite/cc-buglist#537]
<a href="https://github.com/SuddenDeath"><img src="https://avatars1.githubusercontent.com/u/9938017?v=4" align="left" width="96" height="96" hspace="10"></img></a> **Issue by [SuddenDeath](https://github.com/SuddenDeath)**    
_Tuesday Apr 21, 2015 at 19:44 GMT_  
_Originally opened as https://github.com/ccshiro/cc-buglist/issues/537_

----

http://webmup.com/f5809/
http://webmup.com/49768/

watch the pet

**Comment by netherspite** [comment]
<a href="https://github.com/ccshiro"><img src="https://avatars2.githubusercontent.com/u/8135895?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [ccshiro](https://github.com/ccshiro)**  
_Friday May 29, 2015 at 16:17 GMT_

----

We generate our pathing data for WMOs (like the arenas) using recast. We currently don't have a way to tweak it manually when the recast method fails (except changing the generation parameters and passes). In order to allow pathing over ropes we have to remain in this state of pathing in Blade's Edge Arena.

In other words, I can't fix the underlying pathing issue at the moment (without breaking the rope pathing), but I will disable pets being able to cheat their path in arenas. So instead of soaring through the sky it'd just stand still at its position, forcing the warlock to reposition to make it come back to its owner.

That's not an ideal solution, but it's the best we can do without spending significant time on it. Hopefully I can get it fixed better at a later time.
